karma.conf.js

```diff
@@ -69,6 +69,7 @@ const istanbulInstrumenterLoader = {
     /node_modules/,
     /adapter.[jt]s$/,
     /constants.[jt]s$/,
+    /checkbox\/foundation.ts$/,
```

Comment:
Why is this excluded from Istanbul? Are we testing coverage via Jasmine?

Reply:
Yep, since we now have the Jasmine foundation test for checkbox (with coverage for Jasmine tests: https://github.com/material-components/material-components-web/blob/50f110a6cf8100e594bdbd6c02ee278c39924008/karma.conf.js#L132), we can remove test coverage here.

Reply:
Dooooope